db/migrate/20160412024302_devise_create_users.rb

```diff
@@ -0,0 +1,42 @@
+class DeviseCreateUsers < ActiveRecord::Migration[5.0]
+  def change
+    create_table :users do |t|
+      ## Database authenticatable
+      t.string :email,              null: false, default: ""
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.